### ISSUE TITLE
Saved data missing after completeHIT() called

### ISSUE THREAD

**Issue by youngapsmikes** [NYUCCL/psiTurk#416]
Hi, 

I call psiTurk.saveData() in the middle of my experiment and verify (by explicitly looking for the data I've saved in my database) that the data I've saved is stored correctly. However, upon calling psiTurk.completeHIT(), the data that I saw previously is missing. This is on sandbox mode. Does anyone have suggestions?

**Comment by youngapsmikes** [comment]
I think I've identified the issue. For context, I'm using psiturk with heroku. 
I was initially specifying the ad_location parameter improperly in config.txt. When I did this, the HITID would be filled in as debug--- instead of the correct HITID, when on sandbox mode. So I changed the config.txt file accordingly. 

I originally had ad_location = https://myappname.herokuapp.com 
Now I have ad_location = https://myappname.herokuapp.com/ad 

By only making that change, namely appending /ad to ad_location, I now see the missing saved data phenomenon.
